zeebe/engine/src/main/java/io/camunda/zeebe/engine/EngineConfiguration.java

```diff
@@ -13,6 +13,8 @@ public final class EngineConfiguration {
 
   public static final int DEFAULT_MESSAGES_TTL_CHECKER_BATCH_LIMIT = Integer.MAX_VALUE;
   public static final Duration DEFAULT_MESSAGES_TTL_CHECKER_INTERVAL = Duration.ofMinutes(1);
+  public static final int DEFAULT_JOBS_TIMEOUT_TRIGGER_BATCH_LIMIT = Integer.MAX_VALUE;
+  public static final Duration DEFAULT_JOBS_TIMEOUT_POLLING_INTERVAL = Duration.ofSeconds(30);
```

Comment:
I know in the ticket there were comments about using 1s resolution by default, but I was not so sure how I feel about changing the default behavior suddenly.
I'm open for any advice to change it though

Reply:
I think for `main` that's totally ok, for backporting we may keep the old default to not introduce a behavior change

Reply:
IMO, we should resolve the bug on all supported versions. Let's use 1s on all versions by default.